README.md

```diff
@@ -21,6 +21,7 @@ To run DynaFlow launcher, the following environment variables must be defined:
 * IIDM_XML_XSD_PATH: must be the path to xsd of dynawo (generally ${DYNAWO_HOME}/share/iidm/xsd depending on the deployement of Dynawo)
 * DYNAFLOW_LAUNCHER_LOCALE: the locale dictionnary to use. The supported values are the same as for the build. The runtime value can be different from the one used in compilation but all keys must be the same in both files and must use the same number of arguments
 * DYNAFLOW_LAUNCHER_DICTS: the directory where the dictionnaries are located
+* DYNAFLOW_LAUNCHER_PAR: the directory where the constant parameter files are located
```

Comment:
This shouldn't be editable by the user

Reply:
or even visible

Reply:
(this is also true for DYNAFLOW_LAUNCHER_DICTS)

Reply:
This location is not the same if we use the installation or if we use the normal binary so it must be editable. DYNAFLOW_LAUNCHER_DICTS and DYNAFLOW_LAUNCHER_PAR may be fused in DYNAFLOW_LAUNCHER_RESOURCES

Reply:
I would propose DYNAWFLOW_LAUNCHER_INSTALL?

Reply:
DYNAWFLOW_LAUNCHER_INSTALL correspond to CMAKE_PREFIX_INSTALL

Reply:
yes, and user shouldn't have to understand what is put inside this, so this is the only path (along with a distribution) that we can ask him to provide

Reply:
OK